stl/inc/concepts

```diff
@@ -206,9 +206,26 @@ concept _Weakly_equality_comparable_with =
 _EXPORT_STD template <class _Ty>
 concept equality_comparable = _Half_equality_comparable<_Ty, _Ty>;
 
+#if _HAS_CXX23
+template <class _Ty1, class _Ty2, class _Ref = common_reference_t<const _Ty1&, const _Ty2&>>
+concept _Comparison_common_type_with_impl =
+    same_as<_Ref, common_reference_t<const _Ty2&, const _Ty1&>>

```

Comment:
The actual wording in the standard is:

```suggestion
    same_as<common_reference_t<const _Ty1&, const _Ty2&>, common_reference_t<const _Ty2&, const _Ty1&>>
```

however, this also seems to be fine.

Reply:
Yep, this is equivalent. The default template argument is a workaround to simulate having a typedef, which makes the following code clearer by avoiding repetition.